HHS/Head-Start-TTADP
# [Prod] Update R10 goals & objectives, add admin for course management, update requested TRs + sessions

## Description of change
- Close all R10 goals and objectives per user request
- Add admin area for courses
- Mark a requested TR and session as "In progress" per user request

## How to test
View the admin area for courses and confirm that you can view, update, and delete courses
The data requests can be confirmed when the scripts are run on prod.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3123
* https://jira.acf.gov/browse/TTAHUB-2607
* https://jira.acf.gov/browse/TTAHUB-3056

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
